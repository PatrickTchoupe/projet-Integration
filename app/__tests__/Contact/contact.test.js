/* eslint-disable import/imports-first */
/* eslint-disable import/no-unresolved */
/* eslint-disable no-undef */
/* eslint-disable import/no-extraneous-dependencies */
import 'react-native';
import React from 'react';
import Contact from './../../Messenger/contact';
import renderer from 'react-test-renderer';

beforeAll(() => { 
    jest.mock('@react-native-community/async-storage');
  });

it('renders correctly', () => {
    const tree = renderer.create(<Contact />).toJSON();
    expect(tree).toMatchSnapshot();
});
