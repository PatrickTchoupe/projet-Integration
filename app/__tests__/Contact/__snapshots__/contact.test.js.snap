// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "marginTop": 20,
        "width": "100%",
      }
    }
  >
    <Image
      source={
        Object {
          "uri": "undefined",
        }
      }
      style={
        Object {
          "borderRadius": 100,
          "height": 75,
          "width": 75,
        }
      }
    />
    <View
      accessible={true}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "marginLeft": "10%",
          "marginTop": "10%",
          "opacity": 1,
          "width": "100%",
        }
      }
    >
      <Text />
      <Text />
    </View>
    <View />
  </View>
  <View />
  <View
    style={
      Object {
        "alignSelf": "stretch",
        "borderBottomColor": "rgba(177,175,184,0.3)",
        "borderBottomWidth": 0.5,
        "marginTop": 8,
        "width": "100%",
      }
    }
  />
</View>
`;
